sdk/resourcemanager/azure-resourcemanager-containerservice/pom.xml

```diff
@@ -42,6 +42,7 @@
 
       --add-opens com.azure.core/com.azure.core.implementation.util=ALL-UNNAMED
     </javaModulesSurefireArgLine>
+    <revapi.skip>true</revapi.skip>
```

Comment:
What breaks?

Reply:
Please refer the [check results](https://dev.azure.com/azure-sdk/public/_build/results?buildId=2797131&view=results)

Reply:
that is a cancelled pipeline. nothing shows. If you have it, paste here.

Reply:
I can't see the check results. Could you share screenshots?

Reply:
Please refer to the [check results](https://dev.azure.com/azure-sdk/public/_build/results?buildId=2797271&view=results)

Reply:
OK, it should be OK to skip.

Remember, DO NOT skip it, before you have that report, and have it reviewed with reviewers.

Reply:
@XiaofeiCao I am a bit surprised that the break on `ContainerServiceNetworkProfile` does not flag as break on azure-resourcemanager (as KubernetesCluster exposes it as return of a getter).

But if it not get flagged, we would be good.